README.md

```diff
@@ -1,5 +1,11 @@
-# starknet.kt
-StarkNet SDK for Kotlin
+<h2 align="center">☕ starknet jvm ☕</h2>
+
+[StarkNet](https://starkware.co/starknet/#:~:text=Live%20on%20Mainnet-,What%20is%20StarkNet%3F,-StarkNet%20is%20a) SDK for JVM languages:
+- Java
+- Kotlin
+- Scala
+- Clojure
+- Groovy
```

Comment:
I'm not sure if we should include languages we haven't done any testing agains. Or perhaps you've already verified our library works correctly in these?

Reply:
They have java inoperability. It's enough to make sure java version works fine. We have to include them here to make sure typing "Starknet LANG sdk" will make it possible to find our library.

Reply:
Alright, I'll trust you with this one.